dist/angular-google-maps.js

```diff
@@ -6679,13 +6671,15 @@ Original idea from: http://stackoverflow.com/questions/22758950/google-map-drawi
                 map: _gMap
               });
             };
-            if (!_this.validateCoords(scope.center)) {
-              $log.error('angular-google-maps: could not find a valid center property');
+            if (!angular.isDefined(scope.center) && !angular.isDefined(scope.bounds)) {
+              $log.error('angular-google-maps: a center or bounds property is required');
               return;
             }
+            if (!angular.isDefined(scope.center)) {
+              scope.center = new google.maps.LatLngBounds(_this.getCoords(scope.bounds.southwest), _this.getCoords(scope.bounds.northeast)).getCenter();
```

Comment:
Thanks for your reply, 
"if center is not defined set a center" but with scope.bounds properties... what if bound is undefined ?